.github/workflows/python_tests.yml

```diff
@@ -131,6 +131,7 @@ jobs:
     - name: Install Python package
       shell: bash -l {0}
       run: |
+        echo JAVA_HOME=$JAVA_HOME
```

Comment:
@hcho3 @trivialfis 
I am curious for this action where it installs java and which version it uses.